File: src/TmrOverlay.App/Overlays/Standings/StandingsOverlayDefinition.cs
using TmrOverlay.Core.Overlays;

namespace TmrOverlay.App.Overlays.Standings;

internal static class StandingsOverlayDefinition
{
    public static OverlayDefinition Definition { get; } = new(
        Id: "standings",
        DisplayName: "Standings",
        DefaultWidth: 659,
        DefaultHeight: 334,
        FadeWhenLiveTelemetryUnavailable: true);
}

File: local-mac/TmrOverlayMac/Sources/TmrOverlayMac/Overlays/Standings/StandingsOverlayDefinition.swift
import AppKit

enum StandingsOverlayDefinition {
    static let definition = OverlayDefinition(
        id: "standings",
        displayName: "Standings",
        defaultSize: NSSize(width: 620, height: 340),
        fadeWhenLiveTelemetryUnavailable: true
    )
}
